### PR TITLE
CTC import compatibility with tensorflow 0.10

### PR DESCRIPTION
**Pull request by robertomest**
Try except clause to import ctc_loss in new path on tensorflow 0.10.

### CODE REVIEW

**Comment by fchollet** [comment]
This is incomplete: you would also have to modify references to `contrib.ctc` in `ctc_decode()`.

**Comment by robertomest** [comment]
Sorry about that, I didn't realize it because there are no tests for ctc_decode(). After looking into it, I found there are some errors on ctc_decode() which I'm working on fixing. I'm also adding tests for greedy and beam search decoders based on the tensorflow tests. Will update soon.

File: tests/keras/backend/test_backends.py [hunk 1]
```diff
@@ -660,6 +660,110 @@ def test_ctc(self):
...
+            0.584855,  # output beam 0
+            0.389139  # output beam 1
+        ], np.float32)[np.newaxis, :]
+
+        decode_truth = [np.array([1, 0]), np.array([0, 1, 0])]
+
+        beam_width = 2
+        top_paths = 2
+
+        decode_pred_tf, log_prob_pred_tf = KTF.ctc_decode(inputs, input_length, greedy=False, beam_width=beam_width, top_paths=top_paths)
```
Comment (fchollet) [inline tests/keras/backend/test_backends.py]: Please introduce line breaks.

**Comment by fchollet** [comment]
LGTM, thanks!

**Pull request merged**